lib/matplotlib/animation.py

```diff
@@ -1293,14 +1293,30 @@ def _end_redraw(self, evt):
                                                        self._handle_resize)
 
     def to_html5_video(self, embed_limit=None):
-        '''Returns animation as an HTML5 video tag.
+        """
+        Convert the animation to a HTML5 ``<video>`` tag.
```

Comment:
This is still 'an', not 'a'. There is no reading of HTML5 that doesn't start with a vowel sound.